### PR TITLE
fixes for operator flow

### PR DESCRIPTION
**Pull request by kumarabd**
Signed-off-by: kumarabd <abishekkumar92@gmail.com>

**Description**

- Fixed `localhost` connectivity for operator 
- Fixed operator chip and notification message

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: component_info.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "meshery-server",
+  "type": "component",
+  "next_error_code": 1011
```
Comment (leecalcote) [inline component_info.json:4]: Whoa. Awesome. 👏

File: internal/graphql/resolver/error.go [hunk 2]
```diff
@@ -5,66 +5,65 @@ import (
...
 }
 
 func ErrControlPlaneSubscription(err error) error {
-	return errors.NewDefault(ErrControlPlaneSubscriptionCode, "Control Plane Subscription failed", err.Error())
+	return errors.New(ErrControlPlaneSubscriptionCode, errors.Alert, []string{"Control Plane Subscription failed", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrSubscribeChannel(err error) error {
-	return errors.NewDefault(ErrSubscribeChannelCode, "Unable to subscribe to channel", err.Error())
+	return errors.New(ErrSubscribeChannelCode, errors.Alert, []string{"Unable to subscribe to channel", err.Error()}, []string{}, []string{}, []string{})
```
Comment (leecalcote) [inline internal/graphql/resolver/error.go:61]: Where do descriptions, probable cause and suggestion remediation go?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: internal/graphql/resolver/error.go (hunk 2 above)
Comment (kumarabd) [reply internal/graphql/resolver/error.go]: added them now in the code

File: internal/graphql/resolver/error.go [hunk 3]
```diff
@@ -19,51 +19,51 @@ const (
...
-	ErrEmptyHandler   = errors.New(ErrNoMeshSyncCode, errors.Alert, []string{"Database handler not initialized"}, []string{}, []string{}, []string{})
+	ErrNilClient      = errors.New(ErrNilClientCode, errors.Alert, []string{"Kubernetes client not initialized"}, []string{"Kubernetes config is not initialized with Meshery"}, []string{}, []string{"Upload your kubernetes config via the settings dashboard. If uploaded, wait for a minute for it to get initialized"})
+	ErrInvalidRequest = errors.New(ErrInvalidRequestCode, errors.Alert, []string{"Invalid query, please check syntax"}, []string{"The Graphql query requested is invalid"}, []string{}, []string{"Check the query parameters and syntax of the query being run"})
+	ErrNoMeshSync     = errors.New(ErrNoMeshSyncCode, errors.Alert, []string{"MeshSync disabled"}, []string{"MeshSync custom controller is not running in your kubernetes cluster"}, []string{"Meshery Operator is not running in your cluster or is crashing"}, []string{"Enable Meshery Operator from the settings page in the UI", "Check for logs in the meshery-operator pods from inside the application for more information"})
```
Comment (leecalcote) [inline internal/graphql/resolver/error.go:24]: - always capitalize Kubernetes
- "QL" should be capitalized in "Graphql"
-

**Pull request merged**